www/api/ogComponents.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+
+export const OGDocsComponent = ({
+  title,
+  description,
+}: {
+  title: string;
+  description: string;
+}) => {
+  return (
+    <div
+      style={{
+        height: '100%',
+        display: 'flex',
+        width: '100%',
+        flexDirection: 'column',
+        alignItems: 'center',
+        justifyContent: 'center',
+        position: 'relative',
+        fontFamily: 'Inter',
+      }}
+    >
+      <img
+        src="https://assets.trpc.io/www/og-pattern-light.svg"
+        tw="absolute"
+      />
+      <div tw="flex items-center justify-between h-full w-full p-16">
+        <div tw="flex flex-col items-start">
+          <h1 tw="text-6xl text-gray-900">{title}</h1>
+          <p tw="text-2xl text-gray-700">{description}</p>
+        </div>
+
+        <div tw="flex justify-between">
+          <div tw="flex flex-col items-center">
+            <img
+              tw="h-64 w-64 mb-4"
+              src="https://assets.trpc.io/icons/svgs/blue-bg-rounded.svg"
+              alt=""
+            />
+            <span tw="text-5xl font-bold">tRPC</span>
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+export const OGBlogComponent = ({
```

Comment:
I think this can be inlined in the api

Reply:
then that file would have to be tsx and that's not getting deployed.

Reply:
Oh, I missed this, it can't be deployed just because it's `.tsx`, but at the same time `import`ing jsx works?

Reply:
Then I'd say we should move _everything_`*.tsx` file so it's just a slim entrypoint to the tsx file that does all the work

maybe this would work even
```ts
// api/ogBlog.ts

export * from './ogBlogHandler';

// api/ogBlogHandler.tsx

export default /* ...

```